src/styles/main/card.scss

```diff
@@ -0,0 +1,148 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.card {
+  box-sizing: border-box;
+  position: relative;
+  background-color: #fff;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.container {
+  max-width: 1024px;
+  position: relative;
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
+  justify-items: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    margin: 0 auto;
+    position: relative;
+    display: grid;
+    grid-template-columns: 200px 200px;
+    justify-content: center;
+    justify-items: center;
+    gap: 46px 48px;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    margin: 0 auto;
+    position: relative;
+    display: grid;
+    grid-template-columns: 200px 200px 200px;
+    justify-content: center;
+    justify-items: center;
+    gap: 46px 48px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    max-width: 1024px;
+    margin: 0 auto;
+    position: relative;
+    display: grid;
+    grid-template-columns: 200px 200px 200px 200px;
+    justify-content: center;
+    justify-items: center;
+    gap: 46px 48px;
+  }
+}
+
+.content-wrapper {
+  margin: 36px 16px;
+  margin-bottom: 0;
+  height: 148px;
+}
+
+.image {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.product-name {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.product-code {
+  margin-top: 4px;
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  line-height: 14px;
+}
+
+.content-price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.price-text {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+  line-height: 18px;
+}
+
+.prise-num {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.button-link {
+  text-decoration: none;
+}
+
+.button {
+  box-sizing: border-box;
+  padding: 0;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  margin-top: 16px;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background-color: #00acdc;
+  border: none;
+}
+
+.button:hover {
+  border: 1px solid #00acdc;
+  color: #00acdc;
+  background-color: #fff;
+}
+
+.content-revie {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.revie-text {
```

Comment:
There is a typo in the class name `revie-text`. It should be `review-text` to maintain consistency and clarity.